### ISSUE TITLE
Search by Hits

### ISSUE THREAD

**Issue by liamtbrand** [Toxicable/aquarium#11]
- [x] Search should work using number of hits on text in a companies description / title.
- [ ] Companies should be able to be tagged with keywords to bring the most relevant results forward.
- [x] Hits should be counted once.

**Comment by Toxicable** [comment]
@liamtbrand I saw this working on your local version, can you make a PR?

**Comment by liamtbrand** [comment]
Yes, I am just on the bus home, I will do these things once I arrive :)

**Comment by liamtbrand** [comment]
I have merged the changes. It still needs more work to work with descriptions and tags. Currently the search is client side so we should think about moving it to a server if our dataset grows a lot.
